cli/global_state.rs

```diff
@@ -203,7 +204,16 @@ impl GlobalState {
     };
 
     let compiled_module = if was_compiled {
-      state1.ts_compiler.get_compiled_module(&out.url)?
+      state1
+        .ts_compiler
+        .get_compiled_module(&out.url)
+        .map_err(|e| {
+          let msg = e.to_string();
+          OpError::other(format!(
+            "Failed to get compiled source code of {}.\nReason: {}",
+            out.url, msg
+          ))
```

Comment:
If you start adding more messages like this elsewhere, maybe make this into a standalone utility in the future